voila/handler.py

```diff
@@ -65,8 +65,19 @@ def get(self, path=None):
             'theme': self.voila_configuration.theme
         }
 
+        # choose the template
+        template_name = (
+            self.get_query_argument('template', '') or
+            self.voila_configuration.template
+        )
+
+        templates = self.nbconvert_template_paths
+        if self.voila_configuration.template:
+            # prioritize paths based on the template name if specified
+            templates = sorted(self.nbconvert_template_paths, key=lambda p: -(template_name in p))
```

Comment:
This `sorted` call feels bad to me... I suppose we should collect the right template paths for each request. See #258

Reply:
I suppose something cleaner and safer would be `templates = collect_template_paths(..., template_name=...)`

Reply:
Agree it's not very elegant.
Would need to check how nbconvert handles templates that make use of inheritance.